apps/web/src/app/lab/page.tsx

```diff
@@ -12,77 +14,94 @@ export default async function LabPage() {
-          demo="https://regels.overheid.nl/lab/sparql/ui/"
-          docs="https://docs.triply.cc/yasgui/"
-          repo="https://github.com/TriplyDB/Yasgui"
-          licenseVariant="MIT License"
-          license="https://opensource.org/license/mit"
-        />
-        <LabItemCard
-          variant="coming-soon"
-          title="Bold Form Builder"
-          icon="icon-forms"
-          description="Ontwerp conditionele BOLD formulieren sneller en consistenter"
-        />
-        <Card className="flex flex-col items-center justify-center">
-          <Typography variant="large" className="text-center">
-            Wil je jouw eigen tool ook op deze pagina zien?
-          </Typography>
-          <Typography className="text-center">
-            Neem dan contact op met{' '}
-            <Link component="a" href="mailto:steven.gort@ictu.nl" className="underline">
-              Steven Gort.
-            </Link>
-          </Typography>
-        </Card>
-      </Container>
+      <ContainerMethod>
+        <CardLab.Root title="Open GDC Regelregister">
+          De GDC is in essentie een publicatieplatform voor overheden om actieve en passieve openbaarmaking van data,
+          documenten en informatieproducten te ondersteunen. De live demo bevat een prototpe voor Open Regels
+          <CardLab.MethodItems>
+            <CardLab.License label="MIT License" href="https://opensource.org/license/mit" />
+            <CardLab.Demo href="https://regels.dexcat.nl/" />
+            <CardLab.Repo href="https://gitlab.com/xpertselect/portals" />
+            <CardLab.Docs href="http://opengdc.nl" />
+          </CardLab.MethodItems>
+        </CardLab.Root>
+        <CardLab.Root title="Norm Editor" method="FLINT">
+          Norm Editor is een applicatie, gebouwd met behulp van webgebaseerde technologieën, waarmee een gebruiker op
+          een gebruiksvriendelijke en interactieve manier interpretaties van normen in FLINT kan creëren.
+          <CardLab.MethodItems>
+            <CardLab.License
+              label="Apache License 2.0"
+              href="https://gitlab.com/normativesystems/ui/interpretation-editor/-/blob/main/LICENSE?ref_type=heads"
+            />
+            <CardLab.Repo href="https://gitlab.com/normativesystems/ui/interpretation-editor" />
+          </CardLab.MethodItems>
+        </CardLab.Root>
+        <CardLab.Root title="Legal Annotation Tool" method="Wetsanalyse_1">
+          Voor de opleiding HBO-ICT aan de Hogeschool van Amsterdam is door 5 studententeams in één semester gewerkt aan
+          het bouwen van tooling voor Wetsanalyse. De beschikbaar gestelde applicatie is het eindproduct van het team
+          Galactische Zakenpartners dat zonder nadere kwaliteitsslag en/of gebruikerstesten anderszins hier is deployed.
+          <CardLab.MethodItems
+            override={{
```

Comment:
Vraagje: waarom hier een override terwijl ik die bij Norm Editor niet zie? Wat is rationale achter het verschil?